code/modules/mob/mob.dm

```diff
@@ -45,7 +45,8 @@
 	ghostize()
 	if(mind?.current == src) //Let's just be safe yeah? This will occasionally be cleared, but not always. Can't do it with ghostize without changing behavior
 		mind.set_current(null)
-	actions.Cut()
```

Comment:
actions?.cut()

If you are running into a lot of places where you are just making a 1 line nest, it's cleaner to use use abbrv. ? instead